pkg/scripts/download.go

```diff
@@ -44,14 +44,14 @@ func NewDownloader(proxyConfig *api.ProxyConfig) Downloader {
 // Returns information a boolean flag informing whether any download/copy operation
 // happened and an error if there was a problem during that operation
 func (d *downloader) Download(url *url.URL, targetFile string) (*api.SourceInfo, error) {
-	schemeReader := d.schemeReaders[url.Scheme]
+	r := d.schemeReaders[url.Scheme]
```

Comment:
it's not really shadowing anything since they are two different syntactical elements.  and obviously that rule is pretty subjective. (how far away qualifies as further away, and is this really a generic "reader" given that it can hold different types of readers depending on the scheme it's being used for?  all these things make it a little more special imho and deserving of a unique name.